management/server/http/handler/rules.go

```diff
@@ -0,0 +1,128 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/netbirdio/netbird/management/server"
+	"github.com/netbirdio/netbird/management/server/jwtclaims"
+	"github.com/rs/xid"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+// Rules is a handler that returns rules of the account
+type Rules struct {
+	accountManager server.AccountManager
+	authAudience   string
+	jwtExtractor   jwtclaims.ClaimsExtractor
+}
+
+func NewRules(accountManager server.AccountManager, authAudience string) *Rules {
+	return &Rules{
+		accountManager: accountManager,
+		authAudience:   authAudience,
+		jwtExtractor:   *jwtclaims.NewClaimsExtractor(nil),
+	}
+}
+
+// GetAllRulesHandler list for the account
+func (h *Rules) GetAllRulesHandler(w http.ResponseWriter, r *http.Request) {
+	account, err := h.getRuleAccount(r)
+	if err != nil {
+		log.Error(err)
+		http.Redirect(w, r, "/", http.StatusInternalServerError)
+		return
+	}
+
+	writeJSONObject(w, account.Rules)
```

Comment:
Any particular reason why we return a map not a list of rules? 
All other endpoints return [peers] or [users].
This one returns a map. 

```
{
  "ca2im93lo1hs0mlug03g": {
    "ID": "ca2im93lo1hs0mlug03g",
    "Name": "Default",
    "Source": [
      "ca2im93lo1hs0mlug030"
    ],
    "Destination": [
      "ca2im93lo1hs0mlug030"
    ],
    "Flow": 0
  }
}

```

I propose returning rules in such a structure:

```
[
   {
      "ID":"ca2im93lo1hs0mlug03g",
      "Name":"Default",
      "Source":[
         "ca2im93lo1hs0mlug030"
      ],
      "Destination":[
         "ca2im93lo1hs0mlug030"
      ],
      "Flow":0
   }
]
```

It will be much easier to handle on the UI side because all the Table components are built to handle an array.
Also this way we are more consistent with our API

**P.S. The same applies to the /api/groups endpoint**

Reply:
See proposed changes below